src/test/resources/files/complete_holding_record.json

```diff
@@ -4,6 +4,7 @@
   "hrid" : "ho00000000010",
   "holdingsTypeId" : "0c422f92-0f4d-4d32-8cbe-390ebc33a3e5",
   "formerIds" : [ "@former-holding" ],
+  "instanceTitle" : "Sample title;aca08922-a4ac-423e-8e40-065180222761",
   "instanceId" : "aca08922-a4ac-423e-8e40-065180222761",
```

Comment:
Hi @NikitaSedyx , if instanceTitle will have special characters will it be processed correctly? For example : ""Sample; title;aca08922-a4ac-423e-8e40-065180222761"

Reply:
tested with such title

```
m: #as been the industry's stand@rd, dummy text ever < (since- the 15);"


"Lorem ipsum > =~ simp/y dum*y [text]!? -|+ 0f the $pr,{int}ing & 'typ%setting' "industry". \orem ipsum: #as been the industry's stand@rd, dummy text ever < (since- the 15);"
```